src/main.lua

```diff
@@ -321,10 +321,14 @@ end, function(msg)
 		lastKnownWithdrawSupply = LastKnownWithdrawSupply,
 		lastKnownTotalSupply = lastKnownTotalTokenSupply(),
 	}
+
+	-- Stash the formatted from address
+	msg.formattedFrom = utils.formatAddress(msg.From)
```

Comment:
I envisioned this as just replacing the `from` field altogether. And additionally formatting other fields if they exist, e.g.: 
```
msg.tags.Recipient = msg.tags.Recipient and utils.formatAddress(msg.tags.Recipient) or nil
```
and so on for any other address values. I haven't fully thought through whether doing that "destructively" is a bad idea. If we don't want to destroy the incoming value, we can always create more fields as you've done here.

Reply:
34bd2b9

Reply:
<img width="1156" alt="image" src="https://github.com/user-attachments/assets/edf2d6ac-099e-430d-8072-b314a70d76bd">